python/test/unit/hopper/test_persistent_warp_specialized_fused-attention.py

```diff
@@ -321,7 +329,8 @@ def test_op(Z, H, N_CTX, D_HEAD, dtype=torch.float16):
     tri_dq, q.grad = q.grad.clone(), None
     # compare
     assert torch.allclose(ref_out, tri_out, atol=1e-2, rtol=0)
-    assert torch.allclose(ref_dv, tri_dv, atol=1e-2, rtol=0)
+    atol = 4e-2 if is_hip() and get_target()[1] == 'gfx90a' else 1e-2
```

Comment:
Could you also explain the MI200 denorm issue in a comment here? So people don't need to find this PR to learn  the motivation of this change.

Reply:
+1